ick/cmdline.py

```diff
@@ -135,6 +147,9 @@ def run(ctx, dry_run: bool, patch: bool, yolo: bool, filters: list[str]):
                     path.parent.mkdir(parents=True, exist_ok=True)
                     path.write_bytes(mod.new_bytes)
 
+    if json_flag:
+        print(json.dumps(rule_modified, indent=4))
```

Comment:
you should probably put this in a "results" key, so that we can add other info (version? path run on? exit code?) here as well